src/shared/modules/editor/editorDuck.ts

```diff
@@ -108,6 +105,6 @@ export const populateEditorFromUrlEpic = (some$, store) => {
         ]
       }
 
-      return Rx.Observable.of({ type: SET_CONTENT, ...setContent(fullCommand) })
+      return Rx.Observable.of(setContent(fullCommand))
```

Comment:
This was wrong before?

Reply:
I mean, is it an intentional bug fix?

Reply:
The setContent function always returns an object with the type member set to SET_CONTENT so it was redundant to also do it here, typescript complained about it.